### ISSUE TITLE
Accessibility -  live region in Xamarin.Forms ( WAI-ARIA in html)

### ISSUE THREAD

**Issue by elbrinner** [xamarin/Xamarin.Forms#8805]
Imagine that we have a zone or several zones of our App whose content is added, deleted, updated or modified without user intervention.

It is necessary to tell the screen reader what is happening.

Problem, the solutions in IOS is very different from:

Android:
https://codelabs.developers.google.com/codelabs/basic-android-accessibility/#6

UWP:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationlivesetting

IOS:
https://developer.apple.com/documentation/foundation/nsnotificationcenter/1410472-postnotification?language=objc

 
Steps to reproduce.

1) Enable the screen reader.
2.0)  Make a "Pull To Refresh", the screen reader does not receive any indication
2.1) Click on a button that makes a call to a service. (If the service takes 30 seconds) the screen reader receives no indication.
2.2) other cases

Provisional solution applied.

Create a service and verify if the screen reader is activated, if this is the case, send a Toast to the user to notify the screen reader. In the case of IOS, we directly use the PostNotification method

------------------------------------------------

In this demo, I try to simulate one of the problems on button 1, by not notifying the user what is happening on the screen. (when an App is busy and the screen reader does not receive any notification)
https://github.com/elbrinner/accesibilidad-apps-xamarin-forms

**Comment by samhouts** [comment]
@elbrinner I'm not able to compile the sample provided. It's having trouble with the minified.cs. Am I missing something? Thanks!

**Comment by elbrinner** [comment]
@samhouts  

I have updated the example, but it is not complete for UWP. Close the UWP project and compile for IOS or Android. I have downloaded the direct version of the Git to try on another computer and IOS and Android work for me. (The one on android has given me the odd problem, but by restoring the packages and cleaning, I solve the problem)

On the demo:

The "List" button goes to a screen that contains a list, when updating the list, the user is not informed of the change in the view. 

The "List 2" button I have implemented a Service to solve this problem when the screen reader is enabled.

**Comment by elbrinner** [comment]
> @elbrinner I'm not able to compile the sample provided. It's having trouble with the minified.cs. Am I missing something? Thanks!

Actually, you can try the problem with any application.

For Android, "Google Assistent" must be enabled in the accessibility options.

An application that does well is twitter, when doing a “Pull to Refresh” in the list, “Google Assistent” informs the user that the list was updated.

Test the same behavior in an application with Xamarin Forms that has implemented the "Pull to Refresh", the expected behavior should be similar to the application of Twitter, the result is that it does nothing.

The solution I'm applying is in the demo and it works well, I don't know if it's the best way to do it.

**Comment by PureWeen** [comment]
https://github.com/dotnet/maui/issues/3997
